_posts/10-07-2022

```diff
@@ -0,0 +1,34 @@
+---
+title:  "Tableau Tabcmd 2 - Release for general use (Tableau Online functionality)"
+abstract: "The new python tabcmd command-line utility is now available for Tableau Online users"
+---
+
+[On June 10, we published version 1.99.99](https://github.com/tableau/tabcmd/releases/tag/v1.99.99) of the tabcmd command-line utility, for general usage by Tableau Online customers. The 1.99 versioning indicates that it has not yet seen widespread usage, and we expected to find some remaining bugs. 
+
+On September 12, we published an updated [version 2.06](https://github.com/tableau/tabcmd/releases/tag/v2.0.6), that fixed several issues discovered in the 1.99 version and lists a few remaining not-yet-implemented features:
+* when logging in to a server, you must specify http or https with the url and you must specify the exact url (e.g https://10ax.online.tableau.com, not just online.tableau.com)
+* custom views are not supported in any operations
+* get/export size parameters are not respected, and filtering for any parameter with spaces in the names does not work
+* refresh-extracts does not support incremental refresh, asynchronous refresh, or adding/removing calculations for an extract
+* run schedule is not supported
+* a few other options are not yet supported, see release page for details
```

Comment:
It'd be nice to offer a link to the release page. I'm not sure which URL it is though.

Reply:
We'll let Jac confirm but I believe she is referring to the Release Notes section in the main README.md, so we could use this link: https://github.com/tableau/tabcmd#release-notes

Reply:
I was thinking this one - https://github.com/tableau/tabcmd/releases
Any preferences?

Reply:
Yea, pointing to ./releases is probably the best option, will be more future proof